client/components/Landing/LandingStats/LandingStats.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  StatsCardLayout1,
+  StatsCardLayout2,
+  StatsCardLayout3,
+  StatsCardLayout4,
+  StatsCardLayout5,
+  StatsCardLayout6,
+} from '~/components/PlatformStats/v2/StatsCards';
+
+export default {
+  title: 'StatsCards',
+};
+
+export const StatsCard1 = () => (
+  <StatsCardLayout1
+    counter={9}
+    colorText="#2980b9"
+    description="Contribuitori cu inimă mare"
+    src="https://frontend.ro/seo/default-schmoes-seo.jpg"
+  />
+);
+
+export const StatsCard2 = () => (
+  <StatsCardLayout2
+    counter={5}
+    colorText="#A4375B"
+    header="Open-Source"
+    description="Repo-uri"
+    src="https://d3tycb976jpudc.cloudfront.net/public/stats/countries.svg"
+  />
+);
+
+export const StatsCard3 = () => (
+  <StatsCardLayout3
+    text1="Am scris"
+    text2="5 lecții"
+    colorText="#1b1b15"
+    src="https://d3tycb976jpudc.cloudfront.net/public/stats/vintage-typewriter.jpg"
+  />
+);
+
+export const StatsCard4 = () => (
+  <StatsCardLayout4
+    counter1={38}
+    counter2={33}
+    text1="slide-uri"
+    colorText="#1b1b15"
+    text2="challenge-uri practice"
+    title="Am creat și"
+  />
+);
+
+export const StatsCard5 = () => (
+  <StatsCardLayout5
```

Comment:
This card is specific for contributors, as it accepts the `contributors` prop (none other do).

However you didn't add this prop 😅